SeaPublicWebsite.BusinessLogic/Services/QuestionFlowService.cs

```diff
@@ -669,6 +678,16 @@ private QuestionFlowStep FloorInsulatedForwardDestination(PropertyData propertyD
         }
 
         private QuestionFlowStep RoofConstructionForwardDestination(PropertyData propertyData, QuestionFlowStep? entryPoint)
+        {
+            if (entryPoint is not null)
+            {
+                return QuestionFlowStep.AnswerSummary;
+            }
```

Comment:
- Yes, the user is going back to edit
- I've missed some logic here, for the roof construction question
- I've also just realised I've made a mistake with this question placement, because of that. The question before this has follow-on questions (loft questions), and because of the way routing works on FWTS, it will always push the user through the Solar Panels question again to answer the loft questions if they go back and change their answer.

Reply:
I interpreted the ticket as "directly after" the roof construction question, not just after, which was my mistake.

Reply:
spoke with glenn yesterday, we agreed to move the solar pv question to the end of the roof questions flow, which makes the most sense. its also GDS compliant, since putting a non conditional question between conditional ones can cause issues with the change page asking questions it doesnt need to